libs/api/domains/document-provider/src/lib/dto/mutateCategoryOrType.input.ts

```diff
@@ -2,7 +2,7 @@ import { Field, InputType, Int, PartialType } from '@nestjs/graphql'
 import { IsBoolean, IsNumber, IsString } from 'class-validator'
 
 @InputType()
-export class CategoriesAndTypesPostInput {
+export class DocumentProviderCategoriesAndTypesPostInput {
```

Comment:
~ You probably know this but just FYI if you don't want to work with to long class names in code you can always define the GQL schema type in the decorator and have the class name shorter.

```suggestion
@InputType('DocumentProviderCategoriesAndTypesPostInput')
export class CategoriesAndTypesPostInput {
```